src/qp/operators/Distinct.java

```diff
@@ -131,20 +142,36 @@ private boolean checkIfTuplesAreEqual(Tuple firstTuple, Tuple secondTuple) {
         return false;
     }
 
+    /**
+     * Method will check for return value of sorted.close()
+     * @return boolean true or false
```

Comment:
Method is a wrapper for the closing of this and the next operator down the query plan tree